data_structures/src/data_request.rs

```diff
@@ -71,14 +74,30 @@ impl DataRequestPool {
     }
 
     /// Get all the reveals
-    pub fn get_all_reveals(&self) -> HashMap<Hash, Vec<RevealTransaction>> {
+    pub fn get_all_reveals(
```

Comment:
Aside from this function, there is also a `get_reveals` function which still uses the old sort by pkh method.